crates/op-hardforks/src/optimism/mod.rs

```diff
@@ -1 +1,2 @@
 pub(crate) mod mainnet;
+pub(crate) mod sepolia;
```

Comment:
```suggestion
pub mod sepolia;
pub use sepolia::*;
```
let's make this pub and export the constants